lib/fog/aws/compute.rb

```diff
@@ -633,18 +635,12 @@ def _request(body, headers, idempotent, parser, retries = 0)
             raise case match[:code]
                 when 'NotFound', 'Unknown'
                   Fog::Compute::AWS::NotFound.slurp(error, match[:message])
-                when 'RequestLimitExceeded'
-                  if retries < max_retries
-                    jitter = rand(100)
-                    waiting = true
-                    start_time = Time.now
-                    wait_time = ((2.0 ** (1.0 + retries) * 100) + jitter) / 1000.0
+                when 'RequestLimitExceeded'                  
+                  if @retry_if_over_limit && retries < max_retries
+                    jitter = rand * 10 * @retry_jitter_magnitude
```

Comment:
The reworking of this math seems to not want to work out in my brain, so I just want to double check I'm interpreting correctly.

I believe in both cases, it should end up that jitter adds some amount between 0 and 1 second to the resulting value (with the default magnitude). The before/after just change where the multiplication/divisions happen to reach this result. Does that match with your understanding/intention?

Reply:
You're right - 
Original code - the jitter diff 0.01 - 0.1 secs (midpoint 0.05 secs)
An new code - the jitter diff 0.1 - 1 secs (midpoint 0.5 secs) (default)

This makes a bit more sense IMHO given AWS throttling happens at second granularity. 
But happy to change the default to maintain consistency if you prefer?

Ps. I tried to lay it out in such a way that the magnitude makes sense. ie. 
magnitute 0.01 --> jitter midpoint 0.05 secs
magnitute 0.1 (default) --> jitter midpoint 0.5 secs
magnitute 1 --> jitter midpoint 5 secs
magnitute 10 (default) --> jitter midpoint 50 secs
etc.

Reply:
Thanks for laying that out, I thought there might be a difference in values, but my brain was really not cooperating in the specifics. That seems fine, thanks for the detailed explanation.